torchgeo/trainers/byol.py

```diff
@@ -451,3 +451,6 @@ def validation_step(  # type: ignore[override]
         )
 
         self.log("val_loss", loss, on_step=False, on_epoch=True)
+
+    def test_step(self, *args: Any) -> None:  # type: ignore[override]
+        """No-op, does nothing."""
```

Comment:
PyTorch Lightning 1.5 added a feature to check the validity of a trainer. If you pass a DataModule with a `test_loader` but your trainer doesn't have a `test_step`, it raises an error.